docs/manual/DEVELOPER_MANUAL.md

```diff
@@ -100,7 +100,7 @@
             8. [SET.list2set](#implementation-hooks-set-list2set)
         7. [KEQUAL](#implementation-hooks-kequal)
             1. [KEQUAL.eq](#implementation-hooks-kequal-eq)
-            2. [KEQUAL.neq](#implementation-hooks-kequal-neq)
+            2. [KEQUAL.ne](#implementation-hooks-kequal-neq)
```

Comment:
I think we should probably delete this entire file... the hooks section is incomplete, and there are quite a few other sections that (incompletely) repeat content from other files, e.g. the textual `kore` format.